packages/vite/src/plugins/plugin.spec.ts

```diff
@@ -316,6 +316,26 @@ describe('@nx/vite/plugin', () => {
                         "{workspaceRoot}/dist/{projectRoot}",
                       ],
                     },
+                    "dev": {
+                      "command": "vite",
```

Comment:
Maybe we should keep `vite dev`? What happens during `nx init` or `nx import` when we're replacing package scripts?

Reply:
This is going to require some changes to `updatePackageScripts` logic